### ISSUE TITLE
Padding slider?

### ISSUE THREAD

**Issue by lkraav** [Ingvix/sailfishos-patch-improved-notification-preview#3]
Hi. Is it possible to get some spacing inside the box w/ a padding slider control?

**Comment by Ingvix** [comment]
Yes, it is indeed and not that hard to implement either. I've been having trouble with the SDK and not being able to build packages so I can't make any promises but I'm trying to figure it out once again since you made a generous donation. Thanks for that.

**Comment by lkraav** [comment]
Thanks. Drinks on me lol :+1:

**Comment by Ingvix** [comment]
Do you want to control horizontal padding between the edge of the bubble and the notification contents or also between the content items?

**Comment by lkraav** [comment]
I was thinking just bubble edges to start. It may be an iterative process, hard to tell up front.
